frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -10,7 +10,21 @@ Feature: Topology chart area
         @smoke
         Scenario: Empty state of topology : T-01-TC01
              When user navigates to Topology page
-             Then user sees Topology page with message on the top "No resources found"
+             Then user sees Topology page with message "No resources found"
+
+
+        @regression @to-do
+        Scenario: Navigate to Add Empty state of topology
+            Given user navigates to Topology page
+             When user clicks on "Add page" in the topology
+             Then user will be redirected too Add page
+
+
+        @regression @todo
+        Scenario: Add to project option in Empty state of topology
```

Comment:
Instead of this scenario, Can we create one workload using this option?

Reply:
Workload addition from the `Add to project` option is already there in place. Do we need to repeat the process

Reply:
Oh, if it is already in place, then we don't need this scenario